ViewControllers/DetailViewController.swift

```diff
@@ -12,17 +12,18 @@ class DetailViewController: UIViewController {
     static var identifier: String{NSStringFromClass(self)}
     
     var selectedMovieId: Int?
-    
-    let movieDataService = MovieDataService()
-    
     var movieDetails: MovieDetails?
     var languages = [String]()
+    var languagesTest = ["German", "Dutch", "English", "Chinese", "Icelandic", "Russian", "Spanish", "Italian", "Japanese", ]
```

Comment:
I think we are not using this so we can remove it

Reply:
I thought it might come in handy when refactoring the ugly way to display genres and languages.
These properties help to test the implementation.
I'll leave them for now.